data/speakers.ts

```diff
@@ -540,8 +540,7 @@ const speakers: Speaker[] = [
     social: {
       twitter: 'zellwk',
       github: 'zellwk',
-      blog: 'https://medium.com/@zellwk',
-      linkedin: 'https://www.linkedin.com/in/weekeat/',
+      blog: 'https://zellwk.com/blog'
     },
     talks: [],
```

Comment:
Hi @zellwk, thanks for your PR. Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Reply:
Hi, I'm merging this for now, but could you please update your talks in the new PR? Thank you.